src/main/resources/config/application.yml

```diff
@@ -13,6 +13,8 @@ server:
 session.timeout: 3600
 
 spring:
+    main:
+        allow-bean-definition-overriding: true
```

Comment:
Mitkä beanit meillä nyt tarvitsee tämän päälle?

Reply:
Ainakin `MetricConfiguration`in `getMetricRegistry()`